server/core/runtime/plan_step_runner_test.go

```diff
@@ -545,7 +545,7 @@ type remotePlanMock struct {
 	CalledArgs []string
 }
 
-func (r *remotePlanMock) RunCommandAsync(ctx command.ProjectContext, path string, args []string, envs map[string]string, v *version.Version, workspace string) (chan<- string, <-chan runtimemodels.Line) {
+func (r *remotePlanMock) RunCommandAsync(_ command.ProjectContext, _ string, args []string, _ map[string]string, _ *version.Version, _ string) (chan<- string, <-chan runtimemodels.Line) {
```

Comment:
This change looks like it is causing the unit tests to fail: https://github.com/runatlantis/atlantis/actions/runs/5949166065/job/16157295047?pr=3690

Reply:
Locally I get an error here

https://github.com/GMartinez-Sisti/atlantis/blob/7058aa45cfc3fddbafb3e72b8ee57455e63dd14d/server/core/runtime/post_workflow_hook_runner_test.go#L188-L189:

Error:
```sh
                Actual interactions with this mock were:
                SendWorkflowHook(models.WorkflowHookCommandContext{BaseRepo:models.Repo{FullName:"", Owner:"baseowner", Name:"basename", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}, HeadRepo:models.Repo{FullName:"", Owner:"headowner", Name:"headname", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}, Log:(*logging.StructuredLogger)(0x140005163c0), Pull:models.PullRequest{Num:2, HeadCommit:"12345abcdef", URL:"https://github.com/runatlantis/atlantis/pull/2", HeadBranch:"add-feat", BaseBranch:"main", Author:"acme", State:0, BaseRepo:models.Repo{FullName:"", Owner:"", Name:"", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}}, User:models.User{Username:"acme-user"}, Verbose:false, EscapedCommentArgs:[]string(nil), HookID:"", CommandName:"plan"}, "sh: lkjlkj: command not found\r\n", false)
                SendWorkflowHook(models.WorkflowHookCommandContext{BaseRepo:models.Repo{FullName:"", Owner:"baseowner", Name:"basename", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}, HeadRepo:models.Repo{FullName:"", Owner:"headowner", Name:"headname", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}, Log:(*logging.StructuredLogger)(0x140005163c0), Pull:models.PullRequest{Num:2, HeadCommit:"12345abcdef", URL:"https://github.com/runatlantis/atlantis/pull/2", HeadBranch:"add-feat", BaseBranch:"main", Author:"acme", State:0, BaseRepo:models.Repo{FullName:"", Owner:"", Name:"", CloneURL:"", SanitizedCloneURL:"", VCSHost:models.VCSHost{Hostname:"", Type:0}}}, User:models.User{Username:"acme-user"}, Verbose:false, EscapedCommentArgs:[]string(nil), HookID:"", CommandName:"plan"}, "\n", true)
```

Focusing on the end of the output:

```
"sh: lkjlkj: command not found\r\n", false)
"\n", true)
```

Looks somehow related to the issue fixed on https://github.com/runatlantis/atlantis/pull/3425. @X-Guardian you might have more context on this. WDYT?

Reply:
The unit tests are still failing in `plan_step_runner_test.go` same as before.

Reply:
Fixed.